Requirements/Requirements_And_Goals.md

```diff
@@ -0,0 +1,73 @@
+# Requirements and Goals
+
+## Mission
+
+> The Hey, Blue! initiative facilitates moments of meaningful connection between police officers and members of their community by offering intentional opportunities for individuals to meet and share positive experience.
+
+## Goal
+
+To provide a cost effective platform which helps the civilians connect with police officers for a positive interaction experience at a scale in a secure way. This platform can be hosted on both web and mobile based. 
+
+## Actors
+- Citiens
+- Police officers
+- Charities
+- Retail Organisations
+
+## Functional Requirements
+
+### Hard Requirements 
+ 1. End-Use Ease of use (HR1)
```

Comment:
IMO, use of identifiers, e.g. `(HR1)` feels unnecessary

Reply:
agree. i started off and realised its of no use but forgot to delete

Reply:
also who resolves the conversation in these changes? author or collaborator?

Reply:
I think either is fine.